vktrace/vktrace_replay/vkreplay_vkreplay.cpp

```diff
@@ -2855,7 +2495,7 @@ VkResult vkReplay::manually_replay_vkBindBufferMemory(packet_vkBindBufferMemory
             // replayGetBufferMemoryRequirements map.
             VkMemoryRequirements mem_reqs;
             m_vkFuncs.real_vkGetBufferMemoryRequirements(remappeddevice, remappedbuffer, &mem_reqs);
-            replayGetBufferMemoryRequirements[pPacket->buffer] = mem_reqs;
+            replayGetBufferMemoryRequirements[remappedbuffer] = mem_reqs;
         }
         assert(replayGetBufferMemoryRequirements[pPacket->buffer].alignment);
```

Comment:
Seems the key of replayGetBufferMemoryRequirements map should be consistent.  Either use remappedbuffer (e.g. `replayGetBufferMemoryRequirements[remappedbuffer]`) or use the buffer handle in trace file (e.g. `replayGetBufferMemoryRequirements[pPacket->buffer]`).